Staging/EnableGop/EnableGop.c

```diff
@@ -0,0 +1,231 @@
+/** @file
+  Provide GOP on unsupported graphics cards on classic MacPro.
+
+  Copyright (c) 2022-2023, Mike Beaton. All rights reserved.<BR>
+  SPDX-License-Identifier: BSD-3-Clause
+**/
+
+#include <Uefi.h>
+
+#include <Library/DuetBdsLib.h>
+#include <Library/DxeServicesTableLib.h>
+#include <Library/OcBootManagementLib.h>
+#include <Library/OcConsoleLib.h>
+#include <Library/OcDeviceMiscLib.h>
+#include <Library/UefiRuntimeServicesTableLib.h>
+
+#include <Library/OcVariableLib.h>
+
+#define EFI_BLUETOOTH_DELAY_DEFAULT  3000
+
+STATIC EFI_GET_MEMORY_SPACE_MAP  mOriginalGetMemorySpaceMap = NULL;
+
+STATIC
+VOID
+LogStage (
+  CHAR8  Stage
+  )
+{
+  UINT8   One    = 1;
+  CHAR16  Name[] = L"efi-gop-stage0";
+
+  if (FeaturePcdGet (PcdEnableGopDirect)) {
+    Name[6] = L'd';
+  }
+
+  Name[L_STR_LEN (Name) - 1] = Stage;
+  OcSetSystemVariable (
```

Comment:
Should really be debug-only no to wear off NVRAM, to be honest.

Reply:
This is called from a driver in which OcVariableInit is never called ... maybe it should be, or maybe it does not matter ... so this will actually be non-volatile always, I believe.

Is that okay?

Reply:
PS Planning to remove when the driver is a bit more established as working - though I suppose I could make it debug code only and keep it.

Reply:
Maybe really make it DEBUG_CODE_BEGIN/DEBUG_CODE_END? This will be handy…